src/components/ShareIcon/index.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import {Platform, TouchableOpacity} from 'react-native';
 import ShareIOSIcon from '../../assets/icons/images/share.svg';
 import ShareAndroidIcon from '../../assets/icons/images/share-for-android.svg';
-import {COLORS} from '../../utils/theme';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use `colors` instead of `COLORS`. Ensure that the `colors` module exports the required color constants. If not, this could lead to undefined values and potential runtime errors.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->